src/public/sass/UI/product/_modal-form.scss

```
@@ -0,0 +1,109 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.modal-form {
+  display: flex;
+  width: 100%;
+  max-width: 70%;
+  flex-direction: column;
+  align-items: center;
+  margin: 3rem auto;
+
+  &__title {
+    color: v.$blue;
+    font-size: 1.5rem;
```

The size of the titles are different than those of the mockup
```suggestion
    font-size: 2.5rem;
```

You can change to this value and modify de mobile view to make it look like this